### PR TITLE
Fix how policy cards are taken

### PR DESCRIPTION
**Pull request by Terverok**

### CODE REVIEW

File: utils/RoomsManager.js [hunk 1]
```diff
@@ -496,18 +496,24 @@ class RoomsManager {
     getDrawnCards(roomName) {
         return this.rooms_props[roomName].drawnCards
     }
+
+    reShuffle(roomName) {
```
Comment (Terverok) [inline utils/RoomsManager.js]: Just left to write tests for that function

File: utils/__tests__/RoomsManager.spec.js [hunk 2]
```diff
@@ -465,6 +465,17 @@ describe('RoomsManager', () => {
                 [PolicyCards.FacistPolicy, PolicyCards.LiberalPolicy, PolicyCards.LiberalPolicy, PolicyCards.LiberalPolicy],
             )).toEqual(false)
         })
+        const checkIfRoomsCardsMatch = (first, second) => checkIfCardsMatch(
```
Comment (Terverok) [inline utils/__tests__/RoomsManager.spec.js]: Write test for this helper function

File: utils/utils.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const { MessagesTypes } = require('../Dictionary')
 const getCurrentTimestamp = () => Math.floor(Date.now() / 1000)
 
 const log = ({ currentRoom, currentPlayerName }, messageType, message) => {
+    if (process.env.NODE_ENV === 'noConsole') return
```
Comment (Dammic) [inline utils/utils.js]: I would propose to:
- use different env variable, not to overlap the system one,
- add a script to package.json that runs tests without console output, ex test:silent

File: package.json [hunk 4]
```diff
@@ -13,7 +13,8 @@
     "dev": "npm-run-all --parallel build:watch run:dev",
     "eslint": "eslint ./src",
     "build:prod": "rimraf public && webpack --env prod --config webpack.dev.js",
-    "test": "node_modules/jest/bin/jest.js"
+    "test": "node_modules/jest/bin/jest.js",
+    "test:silent": "NODE_SHOWCONSOLE=\"NO\" npm run test"
```
Comment (Dammic) [inline package.json]: better would be 'hideconsole' and then in the utils, check if the variable exists at all.
And true/false would be nicer

**Pull request merged**